code/modules/mob/living/living.dm

```diff
@@ -222,13 +222,20 @@
 		return FALSE
 	if(HAS_TRAIT(src, TRAIT_AI_UNTRACKABLE))
 		return FALSE
-
+	if(user.is_jammed())
+		return FALSE
 	// Now, are they viewable by a camera? (This is last because it's the most intensive check)
 	if(!near_camera(src))
 		return FALSE
 
 	return TRUE
 
+/mob/living/proc/is_jammed()
+	for(var/obj/item/jammer/jammer in GLOB.active_jammers)
+		if(get_dist(get_turf(src), get_turf(jammer)) < jammer.range)
```

Comment:
will it return true if jammer is on another z level?
get_dist says it works through z levels and previously(https://github.com/ParadiseSS13/Paradise/pull/21954) there was an issue with that, i guess here it will be issue too?